### PR TITLE
Docs: Getting started section copy edit

### PR DESCRIPTION
**Pull request by imatwawana**
Style/wording/grammar edits for topics `getting-started.md` and `core-concepts.md`
Also adds some intro text to avoid stacked headings.

@dprokop I've left a couple questions in the text

### CODE REVIEW

File: docusaurus/docs/core-concepts.md [hunk 1]
```diff
@@ -58,17 +60,17 @@ function CounterRenderer({ model }: SceneComponentProps<Counter>) {
...
 interface CounterState extends SceneObjectState {
   count: number;
 }
 ```
 
-### Subscribing to state changes
+### Subscribe to state changes
 
-The component can read the state from scene object by using `model` prop that it receives when rendered. To subscribe to state changes, call `model.useState` method.
+A component can read the state from a scene object by using the `model` property that it receives when rendered. To subscribe to state changes, call the `model.useState` method:
```
Comment (imatwawana) [inline docusaurus/docs/core-concepts.md:73]: I wasn't 100% sure about this edit of changing the first word of the sentence from "the" to "a". This is the first time we're talking about components in this topic, so it seemed like it should be more general on the first instance.

File: docusaurus/docs/getting-started.md [hunk 2]
```diff
@@ -45,9 +49,9 @@ function getScene() {
 }
 ```
 
-### Rendering Scene
+### 2. Render a scene
 
-Use the following code in your Grafana App Plugin page to render the Hello World scene:
+Use the following code in your Grafana App Plugin page to render the "Hello World" scene:
```
Comment (imatwawana) [inline docusaurus/docs/getting-started.md:54]: Should "app plugin" be capitalized in the term "Grafana App Plugin"? I'm not sure if this is a product name or not. Same question for line 10.

Comment (dprokop) [reply docusaurus/docs/getting-started.md:54]: No it shouldn't :) It's just a type of plugin.

File: docusaurus/docs/core-concepts.md (hunk 1 above)
Comment (dprokop) [reply docusaurus/docs/core-concepts.md:73]: Makes sense!

File: docusaurus/docs/core-concepts.md [hunk 3]
```diff
@@ -3,19 +3,21 @@ id: core-concepts
 title: Core concepts
 ---
 
+This page explains the core concepts of Scenes and how to use them in creating your own scene.
```
Comment (imatwawana) [inline docusaurus/docs/core-concepts.md:6]:
```suggestion
This topic explains the core concepts of Scenes and how to use them in creating your own scene.
```

File: docusaurus/docs/getting-started.md [hunk 4]
```diff
@@ -3,21 +3,25 @@ id: getting-started
 title: Set up Scenes
 ---
 
-### Installation
+This page shows you how to install Scenes and create your first "Hello World" scene.
```
Comment (imatwawana) [inline docusaurus/docs/getting-started.md:6]:
```suggestion
This topic describes how to install Scenes and create your first "Hello World" scene.
```

File: docusaurus/docs/getting-started.md [hunk 5]
```diff
@@ -3,21 +3,25 @@ id: getting-started
...
 yarn add @grafana/scenes
 ```
 
-Or use Scene App template to start from scratch: [Create new Scene App](https://github.com/grafana/scenes-app-template/generate)
+Conversely, use the [Scenes app template](https://github.com/grafana/scenes-app-template/generate) to start from scratch.
+
+## Hello World scene
 
-## Hello World Scene
+The following instructions show you how to set up the "Hello World" scene.
```
Comment (imatwawana) [inline docusaurus/docs/getting-started.md:20]:
```suggestion
The following instructions describe how to set up the "Hello World" scene.
```

**Review by dprokop — APPROVED** [review]

**Pull request merged**